docs/kafka/consumer-configuration-kafka/README.adoc

```diff
@@ -74,15 +74,24 @@ NOTE: The Kafka scripts are part of the open source community version of Apache
 endif::[]
 
 //Additional line break to resolve mod docs generation error
+[id="con-consumer-group-list_{context}"]
+== Listing consumer groups
 
+[role="_abstract"]
+If you're using the `rhoas` command-line interface (CLI) to list the consumer groups defined for your Kafka instance use the command:
+
+
+[source,subs="+quotes,+attributes"]
+----
+rhoas kafka consumer-group list
+----
 [id="con-consumer-group-script_{context}"]
 == Connecting the Kafka consumer group script
 
 [role="_abstract"]
 If you're using the `kafka-consumer-groups.sh` script, or any other Kafka scripts,
-you'll need to use the `--bootstrap-server` and `--command-config` flags to connect to your Kafka instance.
+you'll need to use the `--bootstrap-server` and `--command-config` flags to connect to your Kafka instance using the command:
```

Comment:
See https://github.com/redhat-developer/app-services-guides/pull/442/files#r841763723. We should always write in the present tense, unless we're referring explicitly to something that will happen in the future.